packages/twenty-front/src/modules/ui/layout/selectable-list/components/SelectableListItemHotkeyEffect.tsx

```diff
@@ -0,0 +1,19 @@
+import { useSelectableListListenToEnterHotkeyOnItem } from '@/ui/layout/selectable-list/hooks/useSelectableListListenToEnterHotkeyOnItem';
+import { useSelectableListContextOrThrow } from '@/ui/layout/selectable-list/states/contexts/SelectableListContext';
+
+export const SelectableListItemHotkeyEffect = ({
+  itemId,
+  onEnter,
+}: {
+  itemId: string;
+  onEnter: () => void;
+}) => {
+  const { hotkeyScope } = useSelectableListContextOrThrow();
+
+  useSelectableListListenToEnterHotkeyOnItem({
```

Comment:
We should be able to find another solution on a next version but that seems ok for now.